src/tribler/core/utilities/tracker_utils.py

```diff
@@ -90,6 +93,8 @@ def get_uniformed_tracker_url(tracker_url):
 
             if url.scheme == 'http' and uniformed_port == HTTP_PORT:
                 uniformed_url = f'{uniformed_scheme}://{uniformed_hostname}{uniformed_path}'
+            elif url.scheme == 'https' and uniformed_port == HTTPS_PORT:
```

Comment:
NIT: The string `https` is repeated 6 times in this file. Please, consider extracting it to the const.

The same is true for the string `http`.

Reply:
Addressed